src/Setup.h

```diff
@@ -44,6 +47,27 @@ class Setup
     mol.AssignKinds(pdb.atoms, ff);
 
   }
+
+  #if GOMC_LIB_MPI
+  void Init(char const*const configFileName, MultiSim *& multisim)
+  {
+    //Read in all config data
+    config.Init(configFileName, multisim);
+    //Read in FF data.
+    ff.Init(config.in.files.param.name, config.in.ffKind.isCHARMM);
+    //Read PDB data
+    pdb.Init(config.in.restart, config.in.files.pdb.name);
+    //Initialize PRNG
+    prng.Init(config.in.restart, config.in.prng, config.in.files.seed.name);
+
+    //Read molecule data from psf
+    if(mol.Init(config.in.restart, config.in.files.psf.name) != 0) {
+      exit(EXIT_FAILURE);
+    }
+    mol.AssignKinds(pdb.atoms, ff);
+
+  }
+  #endif
```

Comment:
@ynejahi @go2432 Same issue with duplicating the Init function

Reply:
Implemented this like so in header
void Init(char const*const configFileName, MultiSim const*const& multisim)
I used the =NULL default in the Simulation constructor.